docs/products/kafka/kafka-connect/howto/clickhouse-sink-connector.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Create a ClickHouse sink connector for Aiven for Apache Kafka®
+sidebar_label: ClickHouse sink connector
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ConsoleLabel from "@site/src/components/ConsoleIcons";
+
+The ClickHouse sink connector delivers data from Apache Kafka® topics to a ClickHouse database for efficient querying and analysis.
+
+## Prerequisites
+
+Before you begin, ensure that you have the following:
+
+- An [Aiven for Apache Kafka® service](https://docs.aiven.io/docs/products/kafka/kafka-connect/howto/enable-connect)
+  with Apache Kafka Connect enabled or a [dedicated Aiven for Apache Kafka Connect® service](https://docs.aiven.io/docs/products/kafka/kafka-connect/get-started#apache_kafka_connect_dedicated_cluster).
+- Access to a ClickHouse instance with the following connection details:
+  - Hostname, port, and credentials.
+  - A target database and table that have already been created in ClickHouse.
+
+## Limitations
```

Comment:
This section is to be confirmed.
I'll come back to you, I'm trying to test the manual deduplication myself